reconcile/gql_queries/service_dependencies/service_dependencies.py

```diff
@@ -32,6 +32,15 @@
       pipelinesProvider {
         provider
       }
+      resourceTemplates {
+        targets {
+          upstream {
+            instance {
+              name
+            }
+          }
+        }
+      }
```

Comment:
Would it make sense to switch to the new query_string() method?

Reply:
Im not sure I can follow. This file is generated by qenerate.

Reply:
latest qenerate version generates a `query_string` method instead of having a QUERY variable. see https://github.com/chassing/qontract-reconcile/blob/APPSRE-6089__quay_team_creation/reconcile/gql_queries/quay_membership/quay_membership.py#L14

Reply:
We decided against `query_string()` because of potential path mismatches. Depending on how you include the module, the path to the file might not work properly.

The latest qenerate version should not have `query_string` anylonger :thinking:

Reply:
https://github.com/app-sre/qenerate/pull/23

Reply:
Ok, I have an older version installed which still uses the `query_string` method :( Then I'm ok here. I'll reintroduce the `query_string` method in generate, but with a different file path; I like this method :)

Reply:
When you rebase your PR to latest master, then the CI should also fail because if generation mismatch. There was a new CI check added recently https://github.com/app-sre/qontract-reconcile/pull/2663